network-mux/demo/cardano-ping.hs

```diff
@@ -269,8 +302,8 @@ toStatPoint ts host cookie sample td =
     stddev' = fromMaybe 0 (stddev td)
 
 
-pingClient :: Options -> AddrInfo -> IO ()
-pingClient Options{quiet, magic, json} peer = bracket
+pingClient :: (LogMsg -> IO ()) -> Options -> AddrInfo -> IO ()
```

Comment:
Why not just use `Tracer IO LogMsg`?

Reply:
I have no confident in `Tracer`.

Reply:
You are using it!

Reply:
Any similarity with `Tracer` was unintentional.

Reply:
`Tracer` was modelled exactly after this case and  `-> a` is **the cannonical** example of a contravariant functor.

Reply:
I would never have seen it. Will make the adjustment.